main/index.html

```diff
@@ -0,0 +1,1415 @@
+            $("div#Logo").css("padding-top","80px");
+            $("div#Logo img").css("width","400px");
+            }
+            else{
+                $("div#Logo").css("padding-top","0px");
+                $("div#Logo img").css("width","250px");
+            }
+		$(".item").show();
+
+        });
+
+        if(question_asked == 1){
+            get_answer();
+        }
+
+
+
+        $(".infoHeader").hover(function(){
+           var bg = $(this).css("background-image");
+	   bg = bg.replace("images_bw/","images/");
+           $(this).css("background-image",bg);
+        }, function(){
+
+           var bg = $(this).css("background-image");
+	   bg = bg.replace("images/","images_bw/");
+           $(this).css("background-image",bg);
+
+});
+
+});
+
+</script>
+</head>
+<body class="BGribbons">
+<div id="loader" style="height:100%;width:100%;background-color:black; position:fixed;z-index:100;align-content:center;"><img alt="felicity_logo" src="images/felicity_Logo.png"/><br /><br /><img alt="loading" src="images/loalding.gif"/></div>
+
+<div id="fb-root"></div>
+<script>(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "//connect.facebook.net/en_US/all.js#xfbml=1&appId=143418345817318";
+  fjs.parentNode.insertBefore(js, fjs);
+}(document, 'script', 'facebook-jssdk'));</script>
+
+
+
+
+<div id="overlay"></div>
+<div id="lightbulb"></div>
+<div id="container" >
+
+
+	<div id="nav"  class="unit static twoWide threeHigh">
+
+
+<div>
+<iframe style="position:absolute;z-index:10; top:0;left:60px;border-radius:20px;" src='../Misc/sponsors/index.html' height='146' width='272' scrolling='no'></iframe>
+</div>
+
+		<ul class="filter">
+			<li class="selected" id="all" ><a  href="index.html#" data-filter="BADABING"><span>Home</span></a></li>
+			<li id="info"><a href="index.html#"  data-filter=".events"><span>Events</span></a></li>
+			<li id="photography"><a href="index.html#"  data-filter=".about"><span>About</span></a></li>
+			<li id="illustration"><a href="index.html#"  data-filter=".illustration"><span>Schedule</span></a></li>
+			<li id="video"><a href="index.html#"  data-filter=".nights"><span>Nights</span></a></li>
+			<li id="SPM" style="background-color:#558e2f"><a href="index.html#" data-filter=".sponsors"><span>Sponsors</span></a></li>
+		</ul>
+
+	</div>
+	<div class="unit static threeWide" id="social">
+		<ul class="social">
+			<li id="facebook"><a href="https://www.facebook.com/felicity.iiith" target="_blank"></a></li>
+			<li id="twitter"><a href="https://twitter.com/felicity_iiith" target="_blank"></a></li>
+		</ul>
+	</div>
+
+
+<!-- Felicty header -->
+        <div class="unit static threeWide" id="Logo" style="height: 120px">
+
+
+
+
+ <div>
+    <marquee  id="marqueedisp" scrollamount="3" onmouseout="this.scrollAmount=3" onmouseover="this.scrollAmount=0" direction="left" class="list_marquee" >
```

Comment:
I'd say we let them 404, at least for this commit. We'll try to come up with a plan for them later - it's too time consuming and we may not get anywhere finally.